lib/src/main/java/graphql/nadel/NadelTypeDefinitionRegistry.kt

```diff
@@ -12,7 +12,7 @@ import java.util.Collections
  * Alternative to [graphql.schema.idl.TypeDefinitionRegistry] but is more generic
  * and tailored to Nadel specific operations to build the overall schema.
  */
-class NadelDefinitionRegistry {
+class NadelTypeDefinitionRegistry {
```

Comment:
Renamed for clarity as I have also created a `NadelInstructionDefinitionRegistry`